pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -412,30 +434,24 @@ func threadID(didCommMsg *service.DIDCommMsg) (string, error) {
 }
 
 func (s *Service) currentState(thid string) (state, error) {
-	conn, err := s.connectionStore.GetConnectionRecord(thid)
+	connRec, err := s.connectionStore.GetConnectionRecordByNSThreadID(thid)
```

Comment:
rename function parameter to include namespace "ns" in it

Reply:
done